docs/intro.rst

```diff
@@ -0,0 +1,7 @@
+===============
+Getting started
+===============
+
+TODO: Write about how to include Resolwe in a Django project and explain
+settings parameters. Create an example Django project in the ``docs/example``
```

Comment:
Test project is exactly what you described as example project. It supports running Django development server, so you can play with the web UI.